### PR TITLE
[skip treewide] treewide: treefmt with nixfmt

### PR DESCRIPTION
**Pull request by 0x4A6F**
###### Motivation for this change
do not merge [skip ci]

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by 0x4A6F** [comment]
Hitting serokell/nixfmt#81 in `pkgs/development/compilers/nextpnr/default.nix`.

**Comment by SuperSandro2000** [comment]
> Hitting [serokell/nixfmt#81](https://github.com/serokell/nixfmt/issues/81) in `pkgs/development/compilers/nextpnr/default.nix`.

The development is dead for two years so I don't think that will be fixed by upstream.

File: lib/debug.nix [hunk 1]
```diff
@@ -15,27 +15,11 @@
...
-    isDerivation
-    isFunction
-    mapAttrs
-    trace;
-in
-
-rec {
+    isInt attrNames isList isAttrs substring addErrorContext attrValues
+    concatLists concatStringsSep const elem generators head id isDerivation
+    isFunction mapAttrs trace;
```
Comment (SuperSandro2000) [inline lib/debug.nix:20]: This is fully incompatible with nixpkgs-fmt.

File: pkgs/applications/graphics/awesomebump/default.nix [hunk 2]
```diff
@@ -55,7 +55,8 @@ in mkDerivation {
 
   meta = {
     homepage = "https://github.com/kmkolasinski/AwesomeBump";
-    description = "A program to generate normal, height, specular or ambient occlusion textures from a single image";
+    description =
+      "A program to generate normal, height, specular or ambient occlusion textures from a single image";
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/awesomebump/default.nix:59]: This is the kind of formatting why I dislike nixfmt.

**Pull request closed without merging**